cypress/plugins/createRepo.ts

```diff
@@ -9,15 +9,15 @@ const [
   ,
   DOMAIN_NAME = "repository-cypress",
   PASSWORD = process.env.PASSWORD || "",
-  PRISMIC_URL = process.env.PRISMIC_URL,
+  MODE = process.env.MODE,
```

Comment:
This is wrong, you shouldn't have changed it.
The line that calls this is
```
&& npx --yes ts-node ./cypress/plugins/createRepo.ts "${_PRISMIC_REPO}" "${PASSWORD}" "${PRISMIC_URL}"  \
```

No application Mode is passed, we wouldn't have seen the error if the default mode wasn't production.

Reply:
This function should be imported and used.
That way there is no error possible

<img width="677" alt="image" src="https://user-images.githubusercontent.com/41187899/196385175-61295e24-b4be-452a-8765-da9dc4451b76.png">

Reply:
mode is being set here https://github.com/prismicio/slice-machine/pull/643/files#diff-f85a70eb950a383c8d8d55e72cf712c713cc5a4effb291e80a9f471ce783058fR17
maybe he wants to pass it?